### PR TITLE
Adding util to mock out PDF data for report

### PR DESCRIPTION
**Pull request by carakessler**
Adding util that returns fake data (right now just CardData because thats the only type we have) so that we can work on the report independently of the tasks. This is rough for now, will build out and formalize more as we get more types

### CODE REVIEW

File: packages/cli/package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "cli-ux": "^5.4.4",
     "date-and-time": "^0.12.0",
     "devtools-protocol": "^0.0.748504",
+    "faker": "^4.1.0",
```
Comment (mahirshah) [inline packages/cli/package.json]: no `@types/faker`?

File: packages/cli/src/utils/generate-mock-data.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import { CardData, Category, Priority, TaskMetaData } from '@checkup/core';
+/*
+ To use, replace the call for `mergeTaskResults` in `outputResults()` in run.ts with `let resultsForPdf = generateMockData();`
+ You can also comment out the parts of `run()` that actually invoke the tasks to make it faster (since we aren't using the real results anyway)
+**/
```
Comment (scalvert) [inline packages/cli/src/utils/generate-mock-data.ts]: It's not really clear to me what we do with this stuff. Can you actually provide examples?

Comment (scalvert) [reply packages/cli/src/utils/generate-mock-data.ts]: Ping!

File: packages/cli/package.json (hunk 1 above)
Comment (carakessler) [reply packages/cli/package.json]: added

File: packages/cli/src/utils/generate-mock-data.ts (hunk 2 above)
Comment (carakessler) [reply packages/cli/src/utils/generate-mock-data.ts]: i have it working where i just comment out some code and replace existing function call with this one
would you prefer it if (as we dev) `--use-mock`  was a command line option
so we dont need to hackily comment out code?

**Pull request closed without merging**